### DISCUSSION TITLE
Much more awesome website?

### DISCUSSION THREAD

**Discussion by HilbertIdeals5** [mm201/pkmnFoundations#106] [Ideas]
I noticed from the Readme that you're looking for a "much more awesome website". I'm interested in helping, if you're currently considering it.

I'm good with HTML, CSS, JS, PHP, etc. (I write websites from scratch, so no ugly templates), and I'll adapt to whatever data and ideas you have. I also have some brainstorms for designs that are fun, engaging, and reflective of the games, which I'll attach later.

**Comment by mm201** [comment]
I'm curious with what you might come up with, but I'll warn you: I am quite picky.

My intention is to use ASP.NET MVC/Web API and knockout.js. My existing parsers for pkm and other structures are in C# so I really want to go with .NET. If you have a different preference, let me know.

Do you have a Discord?

**Comment by HilbertIdeals5** [comment]
Yes, I have a Discord account.

I was thinking more about the user-facing design of the website, so it would help to know how ASP.NET and knockout.js work, so I can get an idea of how you want the website to operate. A brief explanation and code samples would help.

I'll prepare some sketches and send them soon.

**Comment by HilbertIdeals5** [comment]
Here's a [link](https://drive.google.com/file/d/14Rc3BClGXdqI6neOvSXFoaS2YNuEEbTw/view?usp=sharing) to a rough sketch, to give you an idea of what I'm thinking of. It's a lot, but it's not an impossible mission, at least on the website side.

By the way, how should I give you my Discord? Should I just put it here, or can I email it to you or something?

**Comment by mm201** [comment]
Your best bet is to join [our Discord group](https://discord.com/invite/sDQVKFxvMM). If you just point out who you are, I can give you access to the developer chat.

**Comment by HilbertIdeals5** [comment]
Ok, so I pointed myself out in support.
